Sources/Fluent/Query/QueryField.swift

```diff
@@ -163,3 +163,51 @@ public struct QueryFieldEncodingContainer<Model: Fluent.Model> {
         try container.encode(value, forKey: field)
     }
 }
+
+/// Creating select all field
+extension QueryField {
+    /// Predefined select all field
+    static func all() -> QueryField {
+        return QueryField(name: "*")
```

Comment:
The concept of `*` is SQL-specific. We need something generic.

Reply:
I was giving this only a small chance of success ... I'll remove it :)